src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -167,11 +181,16 @@ protected function getFilterValues(string $filterValues): array
 
     private function isMinRangeSlider(string $name): bool
     {
-        return mb_substr($name, 0, mb_strlen(self::MIN_PREFIX)) === self::MIN_PREFIX;
+        return mb_strpos($name, self::MIN_PREFIX) === 0;
     }
 
     private function isMaxRangeSlider(string $name): bool
     {
-        return mb_substr($name, 0, mb_strlen(self::MAX_PREFIX)) === self::MAX_PREFIX;
+        return mb_strpos($name, self::MAX_PREFIX) === 0;
+    }
+
+    private function isRatingFilter(string $filterName): bool
+    {
+        return $filterName === 'rating';
```

Comment:
Please add `rating` as a constant in this class. Something like `RATING_FILTER_NAME` should be sufficient.

Reply:
Add this constant to the `BaseFilter` class, so it can be used in here and `FINDOLOGIC\FinSearch\Findologic\Response\Xml21\Filter\Filter` respectively.